code/script.js

```diff
@@ -1,9 +1,37 @@
-// A function that adds and remove the class "active" on the section you click on.
-// We haven't really talked about ´this´ yet, but we will... ;)
-// console.log(this) to see how it works
-function toggle() {
-  this.classList.toggle("active")
+window.onscroll = function() {myFunction()};
+
+var header = document.getElementById("hero-image");
+
+var sticky = header.offsetTop;
```

Comment:
I'm guessing that this functionality is making the header sticky to the top on the page - nice touch! One thing that I noticed however was that this may cause the header to be placed above the text in the body, which unfortunately means that the user may have some difficulties to read the content in the FAQ and also to  use the form.  This could be worth looking into a bit more.